csm/core/agent/csm_agent.py

```diff
@@ -120,6 +120,7 @@ def init():
         CsmRestApi._app[const.CSM_USER_SERVICE] = user_service
         CsmRestApi._app[const.STORAGE_CAPACITY_SERVICE] = StorageCapacityService()
         CsmRestApi._app[const.UNSUPPORTED_FEATURES_SERVICE] = UnsupportedFeaturesService()
+        CsmRestApi._app[const.VERSION_VALIDATION_SERVICE] = VersionValidationService()
```

Comment:
Why VersionValidationService? This should be Information service under which version and about information should come

Reply:
Updated